runtime/nls/trace/j9trc.nls

```diff
@@ -99,7 +100,9 @@ J9NLS_TRC_TRACE_WRITE_FAIL_STR=Error writing %d bytes to trace file: %s rc: %d
 J9NLS_TRC_TRACE_WRITE_FAIL_STR.explanation=The specified trace file could not be written to.
 J9NLS_TRC_TRACE_WRITE_FAIL_STR.system_action=The JVM continues but trace data is not written to this file.
 J9NLS_TRC_TRACE_WRITE_FAIL_STR.user_response=If you need to collect the trace data, restart the JVM specifying a trace file that can be opened.
-J9NLS_TRC_TRACE_WRITE_FAIL_STR.sample_input_1=-Xtrace:output=C:\temp\trace.out
```

Comment:
Several other messages inappropriately include `-Xtrace:output=` in samples.
Others have empty samples (e.g. `J9NLS_TRC_MODULE_NOT_LOADED`).
Still other have sample where the template has no parameters (e.g. `J9NLS_TRC_SNAP_WRITE_FAIL`).